.github/workflows/go.yml

```
@@ -0,0 +1,38 @@
+name: Go-CI
+on:
+  push:
+    branches: [ '**' ]
```

```suggestion
    branches: [ master ]
```
otherwise it runs twice when we use our branches on the same repo.